lib/diagnostic.rb

```diff
@@ -106,10 +107,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {:age => 30, :first_name => "fatih"}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
```

Comment:
This doesn't concatenate the value of `:first_name` in the `person` hash and a message as a default value.  `person.default = "#{person[:first_name]} is my first name."`.